canonical-web-and-design/snapcraft.io
# Implement Star Developers

## Done
- Added recognition for star developers using [this](https://app.zeplin.io/project/61532cda7b8d66178a506ae2) design spec.

### Useful links

| Name | Link |
| --------------- | --------------- |
| Store spec | [SN109 - Star Developers](https://docs.google.com/document/d/1Q7ChjpcHyVifo30-QwUC3eAPJFwqRP6r-otJqrrh6xA/edit) |
| Design spec | [WD017 - MP - Implement star developers](https://docs.google.com/document/d/12Oj3dAzc_6VSpWxx9j1beA6bqqjkKklHXb_DaC7NwPw/edit#heading=h.a291qhvyhsw3) |
| Zeplin | https://app.zeplin.io/project/61532cda7b8d66178a506ae2 |

## How to QA
- Visit the demo in your browser, link commented below
> https://snapcraft-io-4028.demos.haus/

1.  Search for 'mvo' in the snap search box

2. Add the string query 'show_star_developer = true' to the url. 

> https://snapcraft-io-4028.demos.haus/search?q=mvo&show_star_developer=true

3. Verify that the star beside the publisher name is visible and up to spec

4. Navigate to the 'josm' snap page.

5. Add the string query 'show_star_developer = true' to the url. 

> https://snapcraft-io-4028.demos.haus/josm?show_star_developer=true

7. Verify that the star beside the publisher name is visible and up to spec

## Screenshots
<img width="1168" alt="Screenshot 2022-06-14 at 9 30 37 AM" src="https://user-images.githubusercontent.com/36668279/173545258-11e170b8-4cf6-4b79-b3be-b1d8715f5f22.png">

<img width="1191" alt="Screenshot 2022-06-14 at 9 30 15 AM" src="https://user-images.githubusercontent.com/36668279/173545328-08208e33-4193-43be-b483-2497e858d473.png">

### Tell Holly once this is done
